src/main/java/com/easypost/model/Order.java

```diff
@@ -20,4 +23,38 @@ public final class Order extends EasyPostResource {
     private Map<String, Object> options;
     private List<ShipmentMessage> messages;
     private List<CarrierAccount> carrierAccounts;
+
+    /**
+     * Get the lowest rate for this Order.
+     *
+     * @return Lowest Rate object
+     * @throws EasyPostException when the request fails.
+     */
+    public Rate lowestRate() throws EasyPostException {
+        return this.lowestRate(null, null);
+    }
+
+    /**
+     * Get the lowest rate for this Order.
+     *
+     * @param carriers The carriers to use in the filter.
+     * @param services The services to use in the filter.
+     * @return Lowest Rate object
+     * @throws EasyPostException when the request fails.
+     */
+    public Rate lowestRate(final List<String> carriers, final List<String> services)
+            throws EasyPostException {
+        return Utilities.getLowestObjectRate(this.getRates(), carriers, services);
```

Comment:
Can we put the base function above the two overloads for this?

Reply:
Sure, I will adjust this in the next PR so I don't dismiss approvals.